src/win/winapi.c

```diff
@@ -109,4 +114,12 @@ void uv_winapi_init() {
     pPowerRegisterSuspendResumeNotification = (sPowerRegisterSuspendResumeNotification)
       GetProcAddress(powrprof_module, "PowerRegisterSuspendResumeNotification");
   }
+
+  secur32_module = LoadLibraryA("secur32.dll");
```

Comment:
Oh, LoL, I did! I'll redo it then :-)

On Aug 22, 2016 14:37, "Kári Tristan Helgason" <notifications@github.com>
wrote:

> In src/win/winapi.c
> <https://github.com/libuv/libuv/pull/834#discussion_r75669008>:
>
> > @@ -109,4 +114,12 @@ void uv_winapi_init() {
> >      pPowerRegisterSuspendResumeNotification = (sPowerRegisterSuspendResumeNotification)
> >        GetProcAddress(powrprof_module, "PowerRegisterSuspendResumeNotification");
> >    }
> > +
> > +  secur32_module = LoadLibraryA("secur32.dll");
>
> You're asking the wrong person I'm afraid 😛 I know absolutely nothing
> about the windows side, IIRC you wrote the windows part of this PR!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/libuv/libuv/pull/834/files/02c174dfd4fea41f18de3940fd889d918d3b352b#r75669008>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AATYGFEvApPSEKORIMf8UoSc3skvp71Cks5qiZgIgaJpZM4IIwU5>
> .
>